app/views/layouts/application.slim

```diff
@@ -127,22 +127,16 @@ html lang='en'
               => icon('fas', 'flag-checkered')
               | Return to race
         header#header.m-1.p-0 = yield(:header)
+        - if current_user && current_user.email.nil?
+          .alert.alert-danger
+            ' You don't have an email attached to your Splits.io account. If
+            ' you become locked out your account will be irrecoverable. You
+            ' will also experience bugs around the site. Please add an email
```

Comment:
I think blatantly stating that you will experience bugs if you don't have a splits.io backed account is pretty strong language.  Maybe can tone this down a bit?

Reply:
Very fair. I'll remove this part.